packages/test/test-utils/package.json

```diff
@@ -104,13 +104,11 @@
   },
   "typeValidation": {
     "version": "2.0.0-internal.2.2.0",
-    "baselineRange": "2.0.0-internal.2.0.0",
+    "baselineRange": ">=2.0.0-internal.2.1.0 <2.0.0-internal.2.2.0",
+    "baselineVersion": "2.0.0-internal.2.1.0",
     "broken": {
       "InterfaceDeclaration_ITestContainerConfig": {
         "backCompat": false
-      },
-      "ClassDeclaration_TestFluidObject": {
-        "forwardCompat": false
```

Comment:
This removal makes sense (together with line 627 in the autogenerated file). Before we cut `internal-2.1.0` I made `TestFluidObject` implement `IFluidRouter`, but there are no further changes between `internal-2.1.0` and `internal-2.2.0`.

Can't speak to the new auto-generated tests though.